src/utilities/__tests__/extendSchema-test.js

```diff
@@ -994,6 +995,16 @@ describe('extendSchema', () => {
     );
   });
 
+  it('maintains configuration of the original schema object', () => {
+    const ast = parse(`
+      extend type Query {
+        __badName: String
+      }
+    `);
+    const schema = extendSchema(testSchema, ast);
+    expect(schema.__allowedLegacyNames).to.eql(['__badName']);
```

Comment:
All other tests are using`to.equal` so I think it should be consistent.

Reply:
I had tried that, but it looks like chai’s `equal` does an identity comparison, whereas `eql` does equality comparison. Seeing as the `__allowedLegacyNames` array gets cloned before passing to the new schema instance, an identity comparison can’t be used here.

Reply:
Did you tried `to.deep.equal`?

Reply:
Ah no, I tried `deepEqual` assuming that Chai was using Node’s `assert` API. Thanks, will try that 👍